app/assets/javascripts/discourse/app/lib/uploads.js

```diff
@@ -354,3 +358,18 @@ export function bindFileInputChangeListener(element, fileCallbackFn) {
   element.addEventListener("change", changeListener);
   return changeListener;
 }
+
+export function formatBytes(bytes, decimals) {
+  if (bytes === 0) {
+    return "0 Bytes";
+  }
+  const kilobyte = 1024,
+    dm = decimals || 2,
+    sizes = ["Bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"],
```

Comment:
Those units should probably be translated as well.

Reply:
https://github.com/discourse/discourse/blob/1472e47aae5bfdfb6fd9abfe89beb186c751f514/app/assets/javascripts/locales/i18n.js#L232-L265

Reply:
Wow I have terrible detective skills, thank you.

Reply:
No worries. I had to look it up as well.

Reply:
https://github.com/discourse/discourse/pull/14415